Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Switchcounter Workshop
+ *
+ */
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '30')),
+    parameters([
+        stringParam(defaultValue: 'http://downloads.creatordev.io/creator-openwrt/v2/pistachio/marduk/creatorci40-v1.1.0-2-rootfs.tar.gz',
+            description: 'Creator image tarball to use', name: "CREATOR_IMAGE"),
+    ])
+])
+
+node('docker && imgtec') {
+    def docker_image
+    docker_image = docker.image "imgtec/workshop-builder:latest"
+    docker_image.inside {
+        stage('Prepare') {
+            checkout([$class: 'GitSCM',
+                userRemoteConfigs: scm.userRemoteConfigs,
+                branches: scm.branches,
+                doGenerateSubmoduleConfigurations: true,
+                submoduleCfg: scm.submoduleCfg,
+                browser: scm.browser,
+                gitTool: scm.gitTool,
+                extensions: scm.extensions + [
+                    [$class: 'CleanCheckout'],
+                    [$class: 'PruneStaleBranch'],
+                    [$class: "RelativeTargetDirectory", relativeTargetDir: "${WORKSPACE}/myworkspace"],
```

Comment:
why not name the target directory with the current repo name itself i.e. w1-switchcounter as this jenkinsfile is specific to this repo?